lambdas/service/app.py

```diff
@@ -860,69 +904,68 @@ def start_manifest_generation():
                     status_code=301 if wait_time else 302)
 
 
-@app.route('/fetch/manifest/files', methods=['GET'], cors=True)
+@app.route('/fetch/manifest/files', methods=['GET'], cors=True, path_spec={
+    'parameters': [
+        params.query(
+            'filters',
+            schema.optional(schema.object(
+                file=schema.object(
+                    **{facet_name: schema.object(**{'is': schema.array(str)})
+                       for facet_name in Plugin.load().service_config().translation.keys()}
+                )
+            )),
+            description='Filters to be applied when generating the manifest',
+        ),
+        params.query(
+            'format',
+            schema.optional(schema.enum('compact', 'full', 'terra.bdbag', type_=str)),
+            description=manifest_description,
+        ),
+        params.query(
+            'token',
+            schema.optional({'type': 'string'}),
+            description='Reserved. Do not pass explicitly.',
+        ),
+    ],
+}, method_spec={
+    'description': unwrap('''
```

Comment:
unwrap() was created for this very purpose. Is the problem that it prevents putting paragraph breaks in the description?

Reply:
And by purpose I mean to be used on `description` fields in specs.

Reply:
I can confirm that the current implementation of the `unwrap` helper does not preserve paragraph breaks. I considered preserving consecutive newlines when writing  but decided not to sink time into adding that feature when my PR had already dragged on for so long. It would not be too difficult to add. The helper is very simple.

Reply:
> but decided not to sink time into adding that feature

It's fine to forgo a change in favour of a simpler implementation that satisfies the current needs. But in this case I would have filed a ticket about this improvement.

> when my PR had already dragged on for so long

That is to be expected. We value quality over quantity because, in the long run, only quality ensures **sustained** productivity.